qiskit_metal/renderers/renderer_elmer/elmer_renderer.py

```diff
@@ -0,0 +1,711 @@
+from typing import Union, List, Tuple, Dict as Dict_
```

Comment:
Thanks Abeer. Is the plan to switch these out all throughout the rest of the repo? I used these types because I noticed that's what's currently being used everywhere else, so I wanted to be consistent.